index.html

```diff
@@ -1 +1,27 @@
 <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Index</title>
+  </head>
+  <body>
+    <main>
+      <section>
+        <p id="pikePlaceStore"></p>
+      </section>
+      <section>
+        <p id="seaTacStore"></p>
+      </section>
+      <section>
+        <p id="southCenterStore"></p>
+      </section>
+      <section>
+        <p id="bellevueSquareStore"></p>
+      </section>
+      <section>
+        <p id="alkiStore"></p>
+      </section>
```

Comment:
Thanks. Wasn't sure before, but after today's code review, it's obvious why I don't need the.